ScienceJournal/ActionArea/ActionAreaBarViewController.swift

```diff
@@ -429,6 +429,12 @@ extension ActionArea {
         (stackView.arrangedSubviews.count ..< 4).forEach { _ in
           stackView.addArrangedSubview(UIView())
         }
+
+        items.forEach { (item) in
+          item.enabler?.observe({ (enabled) in
+            self.setButtonFor(item: item, enabled: enabled)
+          })
```

Comment:
Do all the items need to be re-enabled here? It seems like if one were previously disabled, it may not get turned back on after a button change.